api.json

```diff
@@ -1,55 +1,72 @@
 {
-    "Examples": {
-        "basic": { "sidebarTitle": "Basic Example", "title": "Basic Example" },
-        "refreshing": { "sidebarTitle": "Refreshable ads 1", "title": "Refreshable ads 1", "example": "refreshable" },
-        "refreshing-params": { "sidebarTitle": "Refreshable ads 2", "title": "Refreshable ads 2", "example": "refreshableParams" }
+  "Examples": {
+    "basic": { "sidebarTitle": "Basic Example", "title": "Basic Example" },
+    "refreshing": {
+      "sidebarTitle": "Refreshable ads 1",
+      "title": "Refreshable ads 1",
+      "example": "refreshable"
     },
-    "AdSlot": {
-        "dfpNetworkId": {},
-        "adUnit": {},
-        "slotId": {},
-        "sizes": {},
-        "sizeMapping": {},
-        "adSenseAttributes": {},
-        "targetingArguments": {},
-        "onSlotRender": {},
-        "onSlotIsViewable": {},
-        "onSlotVisibilityChanged": {},
-        "renderOutOfThePage": {},
-        "shouldRefresh": {}
+    "refreshing-params": {
+      "sidebarTitle": "Refreshable ads 2",
+      "title": "Refreshable ads 2",
+      "example": "refreshableParams"
+    }
+  },
+  "AdSlot": {
+    "dfpNetworkId": {},
+    "adUnit": {},
+    "slotId": {},
+    "sizes": {},
+    "sizeMapping": {},
+    "adSenseAttributes": {},
+    "targetingArguments": {},
+    "onSlotRender": {},
+    "onSlotIsViewable": {},
+    "onSlotVisibilityChanged": {},
+    "renderOutOfThePage": {},
+    "shouldRefresh": {}
+  },
+  "DFPSlotsProvider": {
+    "dfpNetworkId": {},
+    "personalizedAds": {},
+    "cookieOption": {},
+    "singleRequest": {},
+    "disableInitialLoad": {},
+    "lazyLoad": {},
+    "limitedAds": {},
```

Comment:
I only added the line ⬆️ 
The rest is just formatting...
I couldn't unstage this changes